package.json

```diff
@@ -35,6 +37,8 @@
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
     "config": "^1.30.0",
+    "angular2-cookie": "^1.2.6",
+    "codeceptjs": "^1.1.5",
```

Comment:
This should be in the `devDependencies` section otherwise it gets included as a transitive dependency if someone pulls the project in.